test/web-platform-tests/to-upstream/html/webappapis/timers/settimeout-setinterval-handles.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<title>Handles returned by setTimeout and setInterval</title>
+<link rel="author" title="Alvaro Dias" href="mailto:alvarorahul@gmail.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#dom-windowtimers-settimeout">
+<link rel="help" href="https://heycam.github.io/webidl/#es-operations">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(() => {
```

Comment:
Should use async_test or promise_test and make sure all the non-cleared callbacks are actually executed (and that none of the cleared ones are executed).